docs/user_guide.md

```diff
@@ -335,14 +360,48 @@ module "forseti" {
       project_id         = "your-forseti-project-id-here"
       org_id             = "your-org-id-here"
       …
-      config_validator_enabled = true
+      config_validator_enabled      = true
+      policy_library_sync_enabled   = true
+      policy_library_repository_url = "git@github.com:forseti-security/policy-library"
     }
 ```
 
-The one important additions is the `config_validator_enabled` field. It is not
-enabled by default; therefore you need to explicitly enable it.
+The important additions are the `config_validator_enabled` and
+`policy_library_sync_enabled` fields. They are not enabled by default; therefore
+you need to explicitly enable them. It is recommended to use the [Policy Library Sync](#policy-library-sync) feature, but not required. If you are using the
+sync, then set `policy_library_repository_url` to the Git URL of your private
+repo.
+
+### Policy Library Sync
+
+Forseti has the ability to clone your private policy library repo and keep it in
+sync (using [git-sync](https://github.com/kubernetes/git-sync)), so that config
+validator will always use the latest policies while scanning. Optionally, you
```

Comment:
This is not optional to use policy-library sync feature right?

Reply:
I don't think it's required. Going forward it will be the only automated way of ensuring Forseti is using the up to date policies, but users could update the policies manually.